src/interface/src/app/home/plan-table/plan-table.component.scss

```diff
@@ -95,3 +95,8 @@ mat-header-row , mat-row{
 .mat-header-cell:not(.mat-column-name ) {
   border-right: 1px solid rgba(0, 0, 0, 0.12);
 }
+
+.spinner {
```

Comment:
Styling here is pretty basic, just centered. We might want to have a fancier loading state but we can improve on this later (unless we think the one we have now is terrible lol)